### PR TITLE
EDEV-104: Wagtail 6.1 upgrade

### PR DESCRIPTION
**Pull request by jamesbiggs**
Ticket URL: [EDEV-104]

## About these changes

Upgraded Wagtail to v6.1 (and a few other patch versions of other packages)

Changed `PASSWORD_REQUIRED_TEMPLATE` to `WAGTAIL_PASSWORD_REQUIRED_TEMPLATE` as required in upgrade considerations.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

[EDEV-104]: https://national-archives.atlassian.net/browse/EDEV-104?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -305,7 +305,7 @@
 
 # Custom password template for private pages
 
-PASSWORD_REQUIRED_TEMPLATE = "password_pages/password_required.html"
+WAGTAIL_PASSWORD_REQUIRED_TEMPLATE = "password_pages/password_required.html"
```
Comment (TNA-Allan) [inline config/settings/base.py:308]: Nice one

**Review by TNA-Allan — COMMENTED** [review]
Query: some mentions of window.chooserUrls deprecation [here ] and some references are made to draftail in etna/core/static/css/wagtail-overrides.css, window.chooserUrls in etna/core/wagtail_hooks.py 
(https://docs.wagtail.org/en/stable/releases/6.1.html#deprecation-of-window-chooserurls-within-draftail-choosers). 
Are we using draftail? Is there any impact on this deprecation?

**Comment by jamesbiggs** [comment]
Hi @TNA-Allan , the places we use chooserUrls are in some JS overrides - I checked these and they're fine 😀

**Review by TNA-Allan — APPROVED** [review]
Looks good to me. Thanks @jamesbiggs

**Review by Firdaussi — APPROVED** [review]
Looks good to me too.

**Pull request merged**